tox.ini

```diff
@@ -19,8 +19,10 @@ commands =
     py.test {posargs}
 
 [testenv:docs]
+deps = -r{toxinidir}/requirements/lint.txt
 changedir = {toxinidir}/docs
 commands =
+    bash -c 'rstcheck `find **/*.rst | xargs`'
```

Comment:
I think we should create a `[testenv:doclint]` with the rstcheck linting process instead of making it part of the `docs` building